dev/devicelab/manifest.yaml

```diff
@@ -584,13 +583,12 @@ tasks:
     stage: devicelab_ios
     required_agent_capabilities: ["mac/ios"]
 
-  # TODO(fujino): does not pass on iOS13 https://github.com/flutter/flutter/issues/41133
-  # system_debug_ios:
-  #   description: >
-  #     Tests that the Engine correctly initializes the system debugger for debug-mode iOS apps.
-  #   stage: devicelab_ios
-  #   required_agent_capabilities: ["mac/ios"]
-  #   timeout_in_minutes: 10
+   system_debug_ios:
```

Comment:
So this is still blocked, but by a different issue. Lemme find it...

Reply:
Can you leave this test commented out, and replace the todo with this issue https://github.com/flutter/flutter/issues/43029